docs/source/installation.rst

```diff
@@ -37,6 +32,13 @@ The `conda-lock` file for OpenFE version v1.0.1 can be downloaded with ::
 
   $ curl -LOJ https://github.com/OpenFreeEnergy/openfe/releases/download/v1.0.1/conda-lock-openfe-1.0.1.yml
 
+We recommend using ``micromamba`` when working with ``conda-lock`` files ::
+
+  $ micromamba create -n openfe --file openfe-1.0.1-conda-lock.yml
+  $ micromamba activate openfe
+
+If ``micromamba`` is not available, ``conda`` or ``mamba`` may be used after installing ``conda-lock``.
```

Comment:
This section is a bit confusing - do you think it would be better if we broke this into two sections? I.e. one "`conda-lock` with micromamba` and `classical `conda-lock`" or something like that?

That way if it doesn't work for one we tell folks to look at the other section.

Reply:
Yes I will re-write it like that, sometimes I don't want to kill my darlings -- but an more thorough rewrite makes more sense